lib/Dialect/FIRRTL/LowerToRTL.cpp

```diff
@@ -996,6 +997,16 @@ LogicalResult FIRRTLLowering::visitExpr(MuxPrimOp op) {
   return setLoweringTo<rtl::MuxOp>(op, ifTrue.getType(), cond, ifTrue, ifFalse);
 }
 
+LogicalResult FIRRTLLowering::visitExpr(ValidIfPrimOp op) {
+  // It isn't clear to me why it it is ok to ignore the binding condition,
+  // but this is what the existing FIRRTL verilog emitter does.
+  auto val = getLoweredValue(op.rhs());
```

Comment:
For posterity, this is [exactly how `RemoveValidIf` works](https://github.com/llvm/circt/pull/212/files#diff-8ca1e3ffddc8ca1645e3129134422712e87b464a617ac1644aadc719d6ac776aR1000).